schedule/yast/autoyast_reinstall@s390x.yaml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
we don't need this file as it is not based on default.

Reply:
fixed.